cubeseed_login/src/component/tables/CertificateTable.tsx

```diff
@@ -0,0 +1,309 @@
+  //       setFarmers(data);
+  //     } catch (error) {
+  //       console.error(error);
+  //     }
+  //   };
+  //   getFarmers();
+  // }, [farmers]);
+
+  const approveDocument = async (farmer: Farmer) => {
+    try {
+      // api endpoint or function passed from parent component here //
+      // functionality for approval email here //
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  const rejectDocument = async (farmer: Farmer) => {
+    try {
+      // api call or function passed from parent component here //
+      // functionality for rejection email here //
+    } catch (error) {
+      console.log(error)
+    }
+  }
+
+  const handlePageClick = (selectedPage: { selected: number }) => {
+    setCurrentPage(selectedPage.selected)
+  }
+
+  const handleButtons = (farmer: Farmer) => {
+    setSelectedFarmer(farmer)
+    setRevealButtons(!revealButtons)
+  }
+
+  return (
+    <div className="holder">
+      <div className="flex items-center">
+        <SearchBar />
+        <Filter />
+      </div>
+      <div className="table">
+        <div className="m-5">
+          <div className="title-row">
+            <p className="ms-1 w-6/12">Full Name</p>
+            <p className="w-5/12">Certificate/Code</p>
+          </div>
+          <div>
+            {displayedFarmers.map((farmer) => (
+              <div key={farmer.id} className="data-row">
+                <div className="flex flex-row">
+                  <p className="ms-2 w-6/12">{farmer.name}</p>
+                  {farmer.certificate ? (
+                    <p className="w-5/12">{farmer.certificate}</p>
+                  ) : (
+                    <p className="w-5/12">{farmer.code}</p>
+                  )}
+                  <button
+                    className="arrow-button"
+                    onClick={() => handleButtons(farmer)}
+                  >
+                    <ChevronDownOutline />
+                  </button>
+                </div>
+                {revealButtons &&
+                  selectedFarmer &&
+                  selectedFarmer.id === farmer.id && (
+                    <div className="button-container">
+                      <button
+                        className="approve-button"
+                        onClick={() => approveDocument(farmer)}
+                      >
+                        <div className="flex items-center">
+                          <CheckmarkCircleOutline />
+                          <p className="ms-2 text-sm ">Approve</p>
+                        </div>
+                      </button>
+                      <button
+                        className="reject-button"
+                        onClick={() => rejectDocument(farmer)}
+                      >
+                        <div className="flex items-center">
+                          <CloseCircleOutline />
+                          <p className="ms-2 text-sm">Reject</p>
+                        </div>
+                      </button>
```

Comment:
- The approve and reject buttons can be a component that you can re-use here and the new farmers table